tasks/api_demo/create_test_data.py

```diff
@@ -0,0 +1,84 @@
+"""
+Task to create some test data using the Data Factory endpoint.
+"""
+
+import json
+
+from screenpy import Actor, Director
+from screenpy.actions import See
+from screenpy_requests.actions import SendPOSTRequest
+from screenpy.pacing import beat
+from screenpy_requests.questions import BodyOfTheLastResponse, StatusCodeOfTheLastResponse
+from screenpy.resolutions import ContainsTheKey, IsEqualTo
+
+from ui.api_demo.device_manager import DATA_FACTORY_URL
+
+from ._base_create_test_data import _BaseCreateTestData
+
+
+class CreateTestData(_BaseCreateTestData):
```

Comment:
This "data factory" task and its base class are cribbed shamelessly from the work of my mentor [@pgoy](https://github.com/perrygoy). Details have been changed to protect the innocent.